@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -231,6 +238,11 @@ export class TileGeometryCreator {
         this.createTextElements(tile, decodedTile, textFilter);
 
         this.createLabelRejectionElements(tile, decodedTile);
+
+        // The ground plane is required for when we change the zoom back and we fall back to the
+        // parent, in that case we reduce the renderOrder of the parent tile and this ground place
+        // ensures that parent doesn't come through.
+        TileGeometryCreator.instance.addGroundPlane(tile);
```

Comment:
There should be a flag in the datasource options weather the plane is added or not. Especially for rendering streets on top of satellite you don't  want to have it.

Reply:
Good point!